### ISSUE TITLE
Update xsd to support CleanTask

### ISSUE THREAD

**Issue by SvenMarcus** [AlamoEngine-Tools/eaw-ci#24]
The `CleanTask` has also been implemented, so we'll need to support that as well in our schema.

Suggestion:
```xml
<Clean>
    <Path>relative/path</Path>
</Clean>
```

**Comment by gruenwaldlk** [comment]
Generation options used:

```
xsd eaw-ci.xsd /c /enableDataBinding /enableLinqDataSet /order /n:EawXBuild.Configuration.v1
```

**Comment by gruenwaldlk** [comment]
Closed via #27
